OneCommunityGlobal/HighestGoodNetworkApp
# Kaushik-Cheng-Yun_WBS.jsx_Unit_test

# Description
Implemented Unit test functions include render WBS and check modal display

## Related PRS (if any):
NA

## Main changes explained:
- Create test file WBS.test.jsx for testing WBS component

## How to test:
1. check into current branch
2. do `npm install` and `npm test src/components/Projects/WBS/wbs.test.jsx` to run this PR locally




## Screenshots or videos of changes:
![Screenshot 2024-04-18 at 12 27 44 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/461ff62c-1953-4bc5-9b10-f9c8283d3ea0)


## Note:
Include the information the reviewers need to know.
